packages/plugins/rubygem-foreman_git_templates/rubygem-foreman_git_templates.spec

```diff
@@ -1,36 +1,41 @@
-# Generated from foreman_git_templates-1.0.0.gem by gem2rpm -*- rpm-spec -*-
-# template: foreman_plugin
```

Comment:
Please keep this line in since it allows auto updating the dependencies when using `bump_rpm.sh`

Reply:
Added the comments with corresponding gem version used for generating the spec-file

Reply:
The `Generated from` line is not that important, but doesn't hurt either.